pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -243,10 +243,11 @@
         },
 
         "ubuntu": {
-            "#import": [ "linux" ]
+            "#import": [ "debian" ]
         },
+
         "ubuntu-x64": {
-            "#import": [ "ubuntu", "linux-x64" ]
+            "#import": [ "ubuntu", "debian-x64" ]
         },
 
         "ubuntu.14.04": {
```

Comment:
Given that we support only selected distro versions the risk that things will go awry on these version is small (once confirmed they work). In the worst case we will have to update the package to contain more libuv,so versions targeted at specific distros/versions.